contentgrid-spring-data-rest/src/test/java/com/contentgrid/spring/data/rest/messages/HalLinkTitlesAndFormPromptsTest.java

```diff
@@ -83,7 +83,64 @@ void promptOnCreateFormPropertiesInHalForms() throws Exception {
                                             type: "text"
                                         },
                                         { name: "content.size", type: "number" }, { name: "content.mimetype", type: "text" }, { name: "content.filename", type: "text" },
-                                        { name: "invoices.number", type: "text" }, { name: "invoices.paid", type: "checkbox" }, { name: "invoices.orders.id" }, { name: "invoices.content.length", type: "number" }
+                                        { name: "invoices.number", type: "text" }, { name: "invoices.paid", type: "checkbox" }, { name: "invoices.orders.id" }, { name: "invoices.content.length", type: "number" },
+                                        {
+                                            name: "sort",
+                                            options: {
+                                                promptField: "prompt",
+                                                valueField: "value",
+                                                inline: [
+                                                    {
+                                                        value: "vat,asc",
+                                                        prompt: "VAT number A🡲Z" # Note that the field value is replaced with the proper translation
```

Comment:
Tests are not updated by commit `Replace arrow with one that is more commonly present in fonts`